### ISSUE TITLE
`sqrt ( 16 )` yields bad error

### ISSUE THREAD

**Issue by blerner** [brownplt/pyret-lang#1110]
First, there's a grammar problem.  Second, it's not terribly informative or actionable.

**Comment by shriram** [comment]
Concretely, it's missing the word "is" (that's the grammar problem). 

But the uninformativeness is a far bigger problem.

**Comment by jswrenn** [comment]
I've corrected the grammatical issue, but the uninformativeness remains. I do not have any direct contact with students, so I do not have any intuition for how this error could be made more actionable. Suggestions?

**Comment by shriram** [comment]
That's a parsing issue that @blerner says is hard to handle.

**Comment by blerner** [comment]
Well, *this particular* example of a *unary* function is tractable: it parses as `id-expr paren-expr` and turns into a well-formedness error that we could actually say "Hey, id-expr paren-expr looks kinda like a function call, so maybe we could give a 'this looks kinda like a function call' error."  But for arity>1 functions, you get a parse error at the first comma, and there's no good way (that I can think of, that's compatible with our grammar and keeping it unambiguous) to handle that parse error.

**Comment by schanzer** [comment]
The current error feels very usable to me

<img width="603" height="222" alt="Image" src="https://github.com/user-attachments/assets/a4bd1ef0-904c-4ef4-9c80-3bea39aaf4dc" />

**Comment by blerner** [comment]
Yes, this unary case is handled. Any higher arity is still tricky.

**Comment by blerner** [comment]
We do have a "Pyret thinks this is likely a function call error".  Closing.
